ui/pages/remote-mode/components/send-allowance-tooltip/send-allowance-tooltip.component.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import {
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../../../components/component-library';
+import Tooltip from '../../../../components/ui/tooltip';
+import {
+  Display,
+  TextColor,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { REMOTE_ROUTE } from '../../../../helpers/constants/routes';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { getIsRemoteModeEnabled } from '../../../../selectors/remote-mode';
+
+type SendAllowanceBannerProps = {
+  hasAllowance: boolean;
+};
+
+export default function SendAllowanceTooltip({
+  hasAllowance,
+}: SendAllowanceBannerProps) {
+  const history = useHistory();
+  const t = useI18nContext();
+  const isRemoteModeEnabled = useSelector(getIsRemoteModeEnabled);
+
+  if (!isRemoteModeEnabled) {
+    return null;
+  }
+
+  return (
+    <Tooltip
+      interactive
+      html={
+        <Box>
+          <Text color={TextColor.textDefault} variant={TextVariant.bodyMd}>
+            {t('sendAllowanceTooltipDescription')}
+            <Text
+              display={Display.Inline}
+              color={TextColor.infoDefault}
+              variant={TextVariant.bodyMd}
+              onClick={() => history.push(REMOTE_ROUTE)}
```

Comment:
If enabled, this link should logically go to /home.html#permissions, although given this page isn’t yet updated with the remote mode section this could potentially come in a later update.